src/Umbraco.Cms.Api.Management/Routing/BackofficeHub.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.SignalR;
+using Umbraco.Cms.Web.Common.Authorization;
+
+namespace Umbraco.Cms.Api.Management.Routing;
+
+[Authorize(Policy = AuthorizationPolicies.BackOfficeAccess)]
+public class BackofficeHub : Hub
+{
+    public async Task SendMessage(string message) => await Clients.All.SendAsync("receiveMessage", message);
```

Comment:
Can this be an object or do the client need to serialize and deserialize?

I wonder if it is better with a JsonElement (or what the system.text.json variant is called) or at least an overload, so there is only a single js method receiving objects